tools/benchmark/.eslintrc.cjs

```diff
@@ -18,7 +18,7 @@ module.exports = {
 	},
 	overrides: [
 		{
-			files: ["src/test/**"],
+			files: ["src/**"],
```

Comment:
This change allows you to take a dependency on node.js in none test code, which is likely to break anyone using this library outside of node.js. I did extra work to ensure this package can be used in environments like browser: I would prefer to not violate that, so I like having this enforcement. While we currently do not support our reporters in those environments, so they disable this lint, I'd like to keep it enforce for all the other code.

Maybe filter this to just ["src/*Reporter.ts", "src/test/**"], and add a comment explaining the reasoning?

Reply:
Revert